### DISCUSSION TITLE
error when run on vs code

### DISCUSSION THREAD

**Discussion by lea1druk** [aldinokemal/go-whatsapp-web-multidevice#121] [General]
C:\Users\nikolov\go\pkg\mod\github.com\h2non\bimg@v1.1.9\image.go:93:49: undefined: Gravity
C:\Users\nikolov\go\pkg\mod\github.com\h2non\bimg@v1.1.9\image.go:133:29: undefined: Watermark
C:\Users\nikolov\go\pkg\mod\github.com\h2non\bimg@v1.1.9\image.go:139:34: undefined: WatermarkImage
C:\Users\nikolov\go\pkg\mod\github.com\h2non\bimg@v1.1.9\image.go:152:26: undefined: Angle
C:\Users\nikolov\go\pkg\mod\github.com\h2non\bimg@v1.1.9\image.go:181:31: undefined: Interpretation
C:\Users\nikolov\go\pkg\mod\github.com\h2non\bimg@v1.1.9\image.go:202:27: undefined: Options
C:\Users\nikolov\go\pkg\mod\github.com\h2non\bimg@v1.1.9\image.go:212:29: undefined: ImageMetadata
C:\Users\nikolov\go\pkg\mod\github.com\h2non\bimg@v1.1.9\image.go:218:35: undefined: Interpretation
C:\Users\nikolov\go\pkg\mod\github.com\h2non\bimg@v1.1.9\image.go:234:25: undefined: ImageSize
C:\Users\nikolov\go\pkg\mod\github.com\h2non\bimg@v1.1.9\resize.go:11:27: undefined: Options
C:\Users\nikolov\go\pkg\mod\github.com\h2non\bimg@v1.1.9\resize.go:11:27: too many errors

**Comment by baharsah** [comment]
hello, there! can you type `go version` on your side?. fyi that works on go 0.21 earlier

**Reply by aldinokemal** [reply]
We no longer use h2non\bimg instead, use the most recent code from master.
